Update tls.md (#5706)
opensearch-project/documentation-website: _install-and-configure/install-dashboards/tls.md

> 

Reviewer: There are some other settings you can find here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/28c1fbb84550d33a036e231661867988af5ebe54/src/cli/serve/serve.js#L102

Which may be worth adding to this doc if not present elsewhere.

Author: added info about more parameters

Setting | Description
:--- | :---

> server.ssl.enabled | Allows for communication between OpenSearch Dashboards and the web browser. Set to `true` for HTTPS or `false` for HTTP.

Author: updated.

Setting | Description
:--- | :---

> server.ssl.enabled | This setting is for communications between OpenSearch Dashboards and the web browser. Set to true for HTTPS, false for HTTP.

Reviewer 2: ```suggestion
server.ssl.enabled | Allows for communication between OpenSearch Dashboards and the web browser. Set to `true` for HTTPS or `false` for HTTP.
```

> server.ssl.certificate | If `server.ssl.enabled` is true, specify the full path to a valid server certificate for OpenSearch Dashboards. You can [generate your own]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/) or get one from a certificate authority.

Reviewer 2: ```suggestion
server.ssl.certificate | If `server.ssl.enabled` is set to `true`, specify the full path to a valid server certificate for OpenSearch Dashboards. You can [generate your own certificate]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/) or get one from a certificate authority.
```

> server.ssl.key | If `server.ssl.enabled` is true, specify the full path (e.g. `/usr/share/opensearch-dashboards-1.0.0/config/my-client-cert-key.pem` to the key for your server certificate. You can [generate your own]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/) or get one from a certificate authority.

Reviewer 2: ```suggestion
server.ssl.key | If `server.ssl.enabled` is set to `true`, specify the full path, for example, `/usr/share/opensearch-dashboards-1.0.0/config/my-client-cert-key.pem`, to the key for your server certificate. You can [generate your own certificate]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/) or get one from a certificate authority.
```

opensearch.ssl.certificateAuthorities | If `opensearch.ssl.verificationMode` is `full` or `certificate`, specify the full path to one or more CA certificates that comprise a trusted chain for your OpenSearch cluster. For example, you might need to include a root CA _and_ an intermediate CA if you used the intermediate CA to issue your admin, client, and node certificates.
opensearch.ssl.alwaysPresentCertificate | Set to `true` to send client certificate towards OpenSearch cluster. It is necessary when mTLS is enabled in OpenSearch. Default is `false`.
opensearch.ssl.certificate | If `opensearch.ssl.alwaysPresentCertificate` is true, specify the full path to a valid client certificate for OpenSearch cluster. You can [generate your own]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/) or get one from a certificate authority.

> opensearch.ssl.key | If `opensearch.ssl.alwaysPresentCertificate` is true, specify the full path (e.g. `/usr/share/opensearch-dashboards-1.0.0/config/my-client-cert-key.pem` to the key for your client certificate. You can [generate your own]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/) or get one from a certificate authority.

Author: Updated

server.ssl.enabled: true
server.ssl.certificate: /usr/share/opensearch-dashboards/config/client-cert.pem
server.ssl.key: /usr/share/opensearch-dashboards/config/client-cert-key.pem

> server.ssl.certificateAuthorities: [ "/usr/share/opensearch-dashboards/config/root-ca.pem", "/usr/share/opensearch-dashboards/config/intermediate-ca.pem" ]

Reviewer: I believe both this and the setting below are currently supported. Not sure why both are implemented but I in the meantime we should probably keep both documented.

Author: opensearch.ssl.certificateAuthorities - CA who signed the server certificate used in OpenSearch cluster
server.ssl.certificateAuthorities - CA who signed the client certificate if mTLS is enabled for Dashboard
